clang/test/SemaSYCL/sycl-include-gnu11.cpp

```diff
@@ -0,0 +1,22 @@
+// RUN: %clang_cc1 -triple spir64-unknown-linux-sycldevice -fcxx-exceptions -fsycl-is-device -verify -fsyntax-only -std=gnu++11 %s -I %S/Inputs
+
+// This test verifies that sycl.hpp can be compiled cleanly with -std=gnu++11
```

Comment:
I would prefer this to committed in a separate commit as this test has noting to do with this PR.

Also this test should be full stack - please, move it to sycl/test and add execution of the compiled app.
This version do not cover the driver functionality.

Reply:
I have removed this test.  I will add it as a separate change.